WakaTime.py

```diff
@@ -298,6 +377,7 @@ def register():
     bpy.app.handlers.load_post.append(load_handler)
     bpy.app.handlers.save_post.append(save_handler)
     bpy.app.handlers.depsgraph_update_pre.append(activity_handler)
+    register_class(WakaTimePreferences)
```

Comment:
Shouldn't it be registered _before_ all the handlers just to be sure that the
`bpy.context.preferences.addons[__name__].preferences`
always returns what you expect it to?

Reply:
Yes, fixed.